mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule PhoenixSwagger.Mixfile do
   use Mix.Project
 
-  @version "0.6.4"
+  @version "0.6.5"
```

Comment:
The reason I chose 0.6.5 is because it's a bug fix, no? It's not introducing new functionality, but fixing existing functionality.